core/src/main/java/hudson/model/AbstractItem.java

```diff
@@ -723,8 +723,10 @@ public static AbstractItem resolveForCLI(
             @Argument(required=true,metaVar="NAME",usage="Job name") String name) throws CmdLineException {
         // TODO can this (and its pseudo-override in AbstractProject) share code with GenericItemOptionHandler, used for explicit CLICommand’s rather than CLIMethod’s?
         AbstractItem item = Jenkins.getInstance().getItemByFullName(name, AbstractItem.class);
-        if (item==null)
-            throw new CmdLineException(null,Messages.AbstractItem_NoSuchJobExists(name,AbstractProject.findNearest(name).getFullName()));
+        if (item==null) {
+            AbstractProject project = AbstractProject.findNearest(name);
+            throw new CmdLineException(null, Messages.AbstractItem_NoSuchJobExists(name, project == null ? "" : project.getFullName()));
```

Comment:
IIRC it has been fixed somewhere in PRs, but probably that one has not been merged.
Makes sense to add something instead of the empty string